pkgs/applications/video/cut-the-crap/default.nix

```diff
@@ -0,0 +1,42 @@
+{ mkDerivation, base, exceptions, fetchgit, generic-lens, hpack
+, hspec, hspec-core, lens, optparse-applicative, optparse-generic
+, regex-tdfa, shelly, stdenv, system-filepath, temporary, text
+, unliftio-core, ffmpeg, makeWrapper
+}:
+(mkDerivation {
+  pname = "cut-the-crap";
+  version = "1.0.0";
+  src = fetchgit {
+    url = "https://github.com/jappeace/cut-the-crap";
+    sha256 = "1z4s0zan421q4722k32p0n33bsh7di1m6pbbdzqi7j87lfd9i7nk";
+    rev = "0ea8a0a0fd3e8d85f44337f13ef2445d22d1bf97";
```

Comment:
Why not use https://github.com/jappeace/cut-the-crap/releases/tag/1.0.0 ?

Reply:
I made this before that debian thing. I wouldn't know how to use tags, or do you just mean the same commit as that tag?

Reply:
Maybe that's also automatically generated...